woostarter/patterns/footer.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Title: footer
+ * Slug: woostarter/footer
+ * Inserter: no
+ */
+?>
+<!-- wp:group {"metadata":{"name":"Default Footer"},"align":"full","className":"is-style-section-1","style":{"spacing":{"padding":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull is-style-section-1" style="padding-top:0;padding-bottom:0"><!-- wp:spacer {"height":"var:preset|spacing|50"} -->
+<div style="height:var(--wp--preset--spacing--50)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|50","left":"var:preset|spacing|50"}}}} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"width":"40%"} -->
+<div class="wp-block-column" style="flex-basis:40%"><!-- wp:heading {"textAlign":"left","level":3,"fontSize":"medium"} -->
+<h3 class="wp-block-heading has-text-align-left has-medium-font-size"><?php esc_html_e('Subscribe to our newsletter', 'woostarter');?></h3>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"align":"left"} -->
+<p class="has-text-align-left"><?php esc_html_e('[Jetpack form]', 'woostarter');?></p>
```

Comment:
I left a question about this Jetpack form here: https://github.com/woocommerce/woo-themes/pull/36#pullrequestreview-2621809795.